chore: Update to .NET 10 RC2

jonpryor opened the pull request:
Context: 41a5796ea0a3a2d270999696ea7e0ce529755e41
Context: 92676ea716a37dfec6786b3fbfd4a8d230f1981f

Looks like .NET 10 RC2 fixed a number of issues which required workarounds (41a5796e, 92676ea7), so remove the temporary workarounds.

**GitHub Issue:** closes #

<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal), unless the change is documentation related. -->

## PR Type:

<!--
Copy the labels that apply to this PR and paste them above:

- 🐞 Bugfix
- ✨ Feature
- 🎨 Code style update (formatting)
- 🔄 Refactoring (no functional changes, no api changes)
- 🏗️ Build or CI related changes
- 📚 Documentation content changes
- 🤖 Project automation
- 💬 Other... (Please describe)

-->


## What is the current behavior? 🤔

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->


## What is the new behavior? 🚀

<!-- Please describe the new behavior after your modifications. -->

## PR Checklist ✅

Please check if your PR fulfills the following requirements:

- [ ] 📝 Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.
- [ ] 🧪 Added [Runtime tests, UI tests, or a manual test sample](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] 📚 Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] 🖼️ Validated PR `Screenshots Compare Test Run` results.
- [ ] ❗ Contains **NO** breaking changes

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->

## Other information ℹ️

<!-- Please provide any additional information if necessary -->

jonpryor committed:
chore: Update to .NET 10 RC2

Context: 41a5796ea0a3a2d270999696ea7e0ce529755e41
Context: 92676ea716a37dfec6786b3fbfd4a8d230f1981f

On Android, .NET 10 RC2 fixed a number of issues which required
workarounds (41a5796e, 92676ea7), so remove the temporary workarounds.

On Apple platforms, the .NET 10 target framework versions changed
from `*18.5` to `*26.0`, e.g. from `net10.0-ios18.5` (.NET 10 RC1)
to `net10.0-ios26.0` (.NET 10 RC2).  Update everything as needed.

Apple platforms also now require Xcode-26.

On iOS, iOS 26 adds a new [`UIView.updateProperties()`][0] method,
which collides with the existing `Pivot.UpdateProperties()` method:

	src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs(120,16):
	  error CS0114: 'Pivot.UpdateProperties()' hides inherited member 'UIView.UpdateProperties()'.
	  To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
	src/SamplesApp/UITests.Shared/Windows_UI_Xaml/XamlRoot/XamlRoot_Properties.xaml.cs(39,15):
	  error CS0114: 'XamlRoot_Properties.UpdateProperties()' hides inherited member 'UIView.UpdateProperties()'.
	  To make the current member override that implementation, add the override keyword. Otherwise add the new keyword. [/Users/runner/work/1/s/src/SamplesApp/SamplesApp.netcoremobile/SamplesApp.netcoremobile.csproj::TargetFramework=net9.0-ios]

Rename the impacted `UpdateProperties()` methods to
`Update<EnclosingType>Properties`() to avoid [warning CS0114][1].

Additionally, `net9.0-ios18.5` no longer exists;
use `net9.0-ios18.0` instead.

Update `linuxVMImage`=[ubuntu-24.04][2] and
`linuxScaledPool`=Ubuntu2404-20251016, as the .NET 10 RC2
Android+Skia+NativeAOT build now requires NDK >= r27, and the
previous images only provided NDK r25.

Update `linux-install-deps.yml` to install `libwebkit2gtk-4.1-dev`,
as `libwebkit2gtk-4.0-dev` doesn't exist in `ubuntu-24.04`.

[0]: https://developer.apple.com/documentation/uikit/uiview/updateproperties()
[1]: https://learn.microsoft.com/dotnet/csharp/misc/cs0114
[2]: https://github.com/actions/runner-images/blob/d4d5736e7d5b88a683d1ea76e7dc74031c71d4c6/images/ubuntu/Ubuntu2404-Readme.md

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21610/wasm-skia-net9/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21610/wasm-skia-net9/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21610/wasm-skia-net9/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21610/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21610/docs/index.html

unodevops commented:
⚠️⚠️ The build [178729](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=178729) has failed on Uno.UI - CI.

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21610/wasm-skia-net9/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-21610/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21610/docs/index.html

jonpryor committed:
chore: attempt to workaround dotnet/aspnetcore#64041

Context: https://github.com/dotnet/aspnetcore/issues/63970
Context: https://github.com/dotnet/aspnetcore/issues/63970#issuecomment-3426101581
Context: https://github.com/dotnet/aspnetcore/issues/64041
Context: https://github.com/dotnet/aspnetcore/issues/64041#issuecomment-3410294048
Context: https://github.com/dotnet/runtime/issues/120790
Context: https://github.com/dotnet/sdk/pull/51269

.NET 10 RC2 broke WebAssembly, causing apps to crash:

	 [MONO] * Assertion at /__w/1/s/src/runtime/src/mono/mono/metadata/assembly.c:2718, condition `<disabled>' not met
	Error
	    at Mc (dotnet.runtime.fn94ls2wwa.js:3:172298)
	    at dotnet.native.ggch313emy.wasm:0xac62
	    at dotnet.native.ggch313emy.wasm:0x43c51
	    at dotnet.native.ggch313emy.wasm:0x42301
	    at dotnet.native.ggch313emy.wasm:0x42422
	    at dotnet.native.ggch313emy.wasm:0x42465
	    at dotnet.native.ggch313emy.wasm:0x42438
	    at dotnet.native.ggch313emy.wasm:0x8cd9
	    at Module._mono_wasm_load_runtime [as mono_wasm_load_runtime] (dotnet.native.98v1chuo8c.js:8:113231)
	    at dotnet.runtime.fn94ls2wwa.js:3:188734
	dotnet.runtime.fn94ls2wwa.js:3
	 MONO_WASM: mono_wasm_load_runtime () failed [object Object]

The fix in dotnet/sdk#51269 is to update
`%(KnownWebAssemblySdkPack.WebAssemblySdkPackVersion)`.
The question is "update *to what*?"  `%(WebAssemblySdkPackVersion)`
refers to the [Microsoft.NET.Sdk.WebAssembly.Pack][0] NuGet package
version, and the only `-rc.2` version doesn't work.  Using the `-rc.1`
version with RC2 *also* doesn't work.

This thought appears unviable: whatever version specified there needs
to be a published NuGet package version, and wherever the fix is, it
doesn't appear to have been published yet.

dotnet/aspnetcore#63970 presents an alternative workaround:

> The issue in .NET 10 RC2 SDK is related to publishing fingerprinted
> assets in WebAssembly based projects. The workaround is to disable
> fingerprinting.

This can be done by settting `$(WasmFingerprintAssets)`=false.

Try it!  Does It Work?

[0]: https://www.nuget.org/packages/Microsoft.NET.Sdk.WebAssembly.Pack

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-21610/docs/index.html

nventive-devops commented:
The [build 179440](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=179440) found UI Test snapshots differences: `android-28-net9`: **19**, `android-28-net9-Snap`: **37**, `ios`: **4**, `ios-Snap`: **53**, `skia-linux-screenshots`: **112**, `skia-windows-screenshots`: **148**, `wasm`: **110**, `wasm-automated-net10.0-WinUI-Benchmarks-automated`: **0**, `wasm-automated-net10.0-WinUI-Default-automated`: **11**, `wasm-automated-net10.0-WinUI-RuntimeTests-0`: **0**, `wasm-automated-net10.0-WinUI-RuntimeTests-1`: **0**, `wasm-automated-net10.0-WinUI-RuntimeTests-2`: **0**

<details>
<summary>Details</summary>


* `android-28-net9`: **19** changed over 825

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Detereminate_ProgressRing_Validation25_[#FF0000_#008000_#FF0000_#FF0000]_Progress-Ring-Value-25`
  - `Detereminate_ProgressRing_Validation50_[#FF0000_#008000_#008000_#FF0000]_Progress-Ring-Value-50`
  - `Detereminate_ProgressRing_Validation75_[#FF0000_#008000_#008000_#008000]_Progress-Ring-Value-75`
  - `FlyoutTest_When_NoOverlayInputPassThroughElement_Then_DontPassThrough_woOff_UITests_Shared_Windows_UI_Xaml_Controls_Flyout_Flyout_OverlayInputPassThroughElement`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `ListView_ListViewWithHeader_InitializesTest_SamplesApp_Windows_UI_Xaml_Controls_ListView_HorizontalListViewGrouped`
  - `NativeCommandBar_Size_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Dynamic`
  - `WebView_NavigateToAnchor_Initial`
  - `When_NoSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `WebView_NavigateToAnchor_Uno_UI_Samples_Content_UITests_WebView_WebView_AnchorNavigation`
  - `When_NoSelection_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `UpDownEnabledTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `When_SingleSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `When_Parent_PointerMoved_After_drag_on_non-scrolling_ScrollViewer`
  - `When_Parent_PointerMoved_After_drag_on_ScrollViewer_-_touch`
  </details>

* `android-28-net9-Snap`: **37** changed over 1075

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RatingControl_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlPage_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlPage`
  - `Pickers_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `Pointers_UITests_Windows_UI_Input_PointersTests_ScrollHandled_UITests_Windows_UI_Input_PointersTests_ScrollHandled`
  - `ListView_ListView_With_ListViews_Count_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `Image_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_Icons_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_Icons`
  - `GridView_Uno_UI_Samples_Content_UITests_GridView_GridView_Item_TextBlock_And_Button_Uno_UI_Samples_Content_UITests_GridView_GridView_Item_TextBlock_And_Button`
  - `CommandBar_Examples_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Examples`
  - `Grid_Grid_with_OutOfRange_Cells_UITests_Shared_Windows_UI_Xaml_Controls_GridTestsControl_Grid_with_OutOfRange_Cells`
  - `MUX_UITests_Shared_Microsoft_UI_Xaml_Controls_TreeViewTests_TreeViewPage_UITests_Shared_Microsoft_UI_Xaml_Controls_TreeViewTests_TreeViewPage`
  - `Scrolling_MUXControlsTestApp_ScrollViewWithScrollControllersPage_MUXControlsTestApp_ScrollViewWithScrollControllersPage`
  - `Icons_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground`
  - `Image_LargeImageList_Uno_UI_Samples_UITests_LargeImageList`
  - `WebView_Javascript_AlertConfirmPrompt_Uno_UI_Samples_Content_UITests_WebView_WebViewControlJavaScriptAlertConfirmPrompt`
  - `WebView_WebView_ChromeClient_Uno_UI_Samples_Content_UITests_WebView_WebView_ChromeClient`
  - `WebView_WebView_JavascriptInvoke_Uno_UI_Samples_Content_UITests_WebView_WebView_JavascriptInvoke`
  - `WebView_WebView_Mailto_Uno_UI_Samples_Content_UITests_WebView_WebView_Mailto`
  - `SwipeControl_MUXControlsTestApp_SwipeControlPage2_MUXControlsTestApp_SwipeControlPage2`
  - `TextBlock_UITests_Shared_Windows_UI_Xaml_Controls_TextBlockControl_TextBlock_Layout_UITests_Shared_Windows_UI_Xaml_Controls_TextBlockControl_TextBlock_Layout`
  - `WebView_WebView2_Mailto_SamplesApp_Microsoft_UI_Xaml_Controls_WebView2Tests_WebView2_Mailto`
  </details>

* `ios`: **4** changed over 255

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `TextBox_UpdatedBinding_On_OneWay_Mode_UITests_Windows_UI_Xaml_Controls_TextBox_TextBox_Bindings`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  </details>

* `ios-Snap`: **53** changed over 992

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Grid_CenteredGridinGridwiththreefixedsizechildren_Uno_UI_Samples_Content_UITests_GridTestsControl_CenteredGridinGridwiththreefixedsizechildren`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Horizontal_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Horizontal`
  - `Microsoft_UI_Composition_UITests_Windows_UI_Composition_InteractionTrackerAndExpressionAnimationSample_UITests_Windows_UI_Composition_InteractionTrackerAndExpressionAnimationSample`
  - `Microsoft_UI_Composition_UITests_Windows_UI_Composition_MixTransformCliPropertyAndClippedByParentWithBorders_Then_RenderingIsValid_UITests_Windows_UI_Composition_MixTransformCliPropertyAndClippedByParentWithBorders_Then_RenderingIsValid`
  - `Brushes_ImageBrushStretch2_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushStretch2`
  - `Icons_UITests_Microsoft_UI_Xaml_Controls_ImageIconTests_ImageIconPage_UITests_Microsoft_UI_Xaml_Controls_ImageIconTests_ImageIconPage`
  - `Icons_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground`
  - `ListView_ListView_Aligned_Left_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_Aligned_Left`
  - `Microsoft_UI_Xaml_Media_CompositionTarget_Rendering_UITests_Shared_Windows_UI_Xaml_Media_CompositionTargetTests_CompositionTarget_Rendering`
  - `TextBlock_TextBlock_FixedWidth_With_DataBound_Run_Uno_UI_Samples_Content_UITests_TextBlockControl_TextBlock_FixedWidth_With_DataBound_Run`
  - `Brushes_UITests_Windows_UI_Xaml_Media_BrushesTests_RevealBrush_Fallback_UITests_Windows_UI_Xaml_Media_BrushesTests_RevealBrush_Fallback`
  - `ListView_ListView_With_ListViews_Count_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `NavigationView_MUXControlsTestApp_NavigationViewRS3Page_MUXControlsTestApp_NavigationViewRS3Page`
  - `ToggleSwitch_Native_ToggleSwitch_IsOn_SamplesApp_Windows_UI_Xaml_Controls_ToggleSwitchControl_Native_ToggleSwitch_IsOn`
  - `Windowing_UITests_Windows_UI_Xaml_WindowTests_Window_SetBackground_UITests_Windows_UI_Xaml_WindowTests_Window_SetBackground`
  - `WebView_WebView_JavascriptInvoke_Uno_UI_Samples_Content_UITests_WebView_WebView_JavascriptInvoke`
  - `GridView_GenericApp_Views_Samples_Shared_Content_UITests_GridViewGrouped_GenericApp_Views_Samples_Shared_Content_UITests_GridViewGrouped`
  - `Windows_ApplicationModel_Contacts_Pick_UITests_Windows_ApplicationModel_Contacts_PickContact`
  - `Brushes_PanelImageBrush_Uno_UI_Samples_UITests_ImageBrushTestControl_PanelImageBrush`
  - `Image_Uno_UI_Samples_UITests_Image_Image_Stretch_Alignment_Bigger_Uno_UI_Samples_UITests_Image_Image_Stretch_Alignment_Bigger`
  </details>

* `skia-linux-screenshots`: **112** changed over 2292

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ButtonClippingTestsControl.png`
  - `Buttons.png-dark`
  - `DisplayInformation.png-dark`
  - `DisplayInformation.png`
  - `BitmapIcon_Monochromatic.png`
  - `ImageBrush_Stretch.png-dark`
  - `ImageBrush_Stretch.png`
  - `Gamepad_CurrentReading.png-dark`
  - `Gamepad_CurrentReading.png`
  - `ImageBrushInList.png-dark`
  - `ImageBrushInList.png`
  - `ImageBrushLocalAsset.png-dark`
  - `ImageBrushLocalAsset.png`
  - `Gamepad_Enumeration.png-dark`
  - `Gamepad_Enumeration.png`
  - `Grid_with_UILabel_TextAlignmentVertical_Bottom.png-dark`
  - `Grid_with_UILabel_TextAlignmentVertical_Bottom.png`
  - `CalendarView_Theming.png-dark`
  - `CalendarView_Theming.png`
  - `EllipsemaskingEllipseGrid.png-dark`
  </details>

* `skia-windows-screenshots`: **148** changed over 2292

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Border_With_ScaleTransform.png`
  - `Border_With_TranslateTransform.png-dark`
  - `DisplayInformation.png-dark`
  - `DisplayInformation.png`
  - `Gamepad_CurrentReading.png-dark`
  - `Gamepad_CurrentReading.png`
  - `Basics.png-dark`
  - `Border_With_RotateTransform.png`
  - `Border_With_ScaleTransform.png-dark`
  - `Grid_With_RotateTransform_And_Button.png-dark`
  - `Grid_With_RotateTransform_And_Button.png`
  - `Buttons.png`
  - `EllipsePage.png-dark`
  - `EllipsePage.png`
  - `Border_With_Off_Centre_RotateTransform.png`
  - `Border_With_Off_Centre_ScaleTransform.png-dark`
  - `DoubleAnimation_Opacity_TextBlock.png-dark`
  - `DoubleAnimation_Opacity_TextBlock.png`
  - `DoubleAnimation_RenderTransformOrigin.png-dark`
  - `DoubleAnimation_RenderTransformOrigin.png`
  </details>

* `wasm`: **110** changed over 1056

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `UITests.Microsoft_UI_Xaml_Controls.RefreshContainerTests.RefreshContainerScrollTop`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Ogg_Extension`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MenuFlyoutTests.MenuFlyoutItem_Click`
  - `UITests.Shared.Windows_UI_Xaml_Controls.WebView.WebView_NavigateToString2`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListView_DisplayMemberPath`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListView_DuplicateItem`
  - `UITests.Windows_UI_Xaml_Controls.TextBox.TextBox_VerticalAlignment`
  - `UITests.Windows_UI_Xaml_Controls.TextBox.TextBox_Visibility`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListViewHeaderFooter_ImplicitlyBinded`
  - `UITests.Shared.Windows_UI_Xaml_Controls.Button.Button_Events`
  - `UITests.Shared.Windows_UI_Xaml_Controls.Popup.MessageDialog`
  - `UITests.Windows_UI_Xaml.DragAndDrop.DragDrop_TestPage`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListViewItem_IsEnabled`
  - `UITests.Windows_UI_Xaml_Controls.TextBlockControl.TextBlock_LayoutAlignment`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Changing_Text`
  - `UITests.Microsoft_UI_Xaml_Controls.ScrollViewTests.KeyboardNavigationSample`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ContentDialogTests.ContentDialog_Closing`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ContentDialogTests.ContentDialog_Simple`
  - `UITests.Shared.Windows_UI_Xaml_Controls.Slider.Slider_Features`
  - `UITests.Uno_Web.Http.CookieManagerTests`
  </details>

* `wasm-automated-net10.0-WinUI-Benchmarks-automated`: **0** changed over 1

* `wasm-automated-net10.0-WinUI-Default-automated`: **11** changed over 877

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `Default_StrokeThickness_MyLine`
  - `TestProgressRing_InitialState_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `Default_StrokeThickness_MyPolygon`
  - `Default_StrokeThickness_MyPath`
  - `Default_StrokeThickness_MyRect`
  - `Default_StrokeThickness_MyEllipse`
  - `When_Theme_Changed_No_Crash_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `Default_StrokeThickness_MyPolyline`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  </details>

* `wasm-automated-net10.0-WinUI-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net10.0-WinUI-RuntimeTests-1`: **0** changed over 1

* `wasm-automated-net10.0-WinUI-RuntimeTests-2`: **0** changed over 1

</details>
